modules/andes-core/broker/src/main/java/org/wso2/andes/mqtt/MQTTLocalSubscription.java

```diff
@@ -235,6 +235,11 @@ public long getSubscribeTime() {
         return 0;
     }
 
+    @Override
+    public String getProtocolQueueName() {
+        return wildcardDestination;
```

Comment:
This is implemented just to return the MQTT topic name. Anyway, protocolQueue name only used in AMQP non durable topic subscription details.